app/controllers/articles_controller.rb

```diff
@@ -84,6 +84,16 @@ def destroy
     redirect_to :articles
   end
 
+  def destroy_all
+    @article = current_user.articles
```

Comment:
We don't need to use an instance variable here as we don't need to expose the user's articles to the view - a regular variable would do:

```suggestion
    articles_to_destroy = current_user.articles
```